### ISSUE TITLE
Blackmagic ATEM gives a constant "connecting" status but never actually connects

### ISSUE THREAD

**Issue by assassin4438** [bitfocus/companion-module-bmd-atem#213]
I saw that there was another issue similar to this, and I've updated bitfocus to the latest version. I am currently running 2.3.1 (2.3.1+4641-v2-3.1-dc01ac7c). This issue started a little over a week ago and i have been running myself ragged trying to figure out what the heck is going on. anyone have any ideas?

**Comment by pixelauthor** [comment]
I am now experiencing the same issue as you were. Did you ever find a solution? Updated to the most recent stable release (2.4.2+4911-v2-4.2-fcb5a863) but cannot get the ATEM to connect. It can obviously see the ATEM because if I change the name or IP using ATEM setup, it updates.

**Comment by assassin4438** [comment]
> I am now experiencing the same issue as you were. Did you ever find a solution? Updated to the most recent stable release (2.4.2+4911-v2-4.2-fcb5a863) but cannot get the ATEM to connect. It can obviously see the ATEM because if I change the name or IP using ATEM setup, it updates.

yes actually, for me it ended up being an issue on my network configuration where the devices on the network switch the ATEM was connected to, and the devices on the network switch the computer was connected to weren't allowed to talk to each other.  In my case, it was due to the fact that I am using Dante for my sound production, and my network guys wanted to separate the 2. I'm not sure exactly what they did, but they made it so the 2 could talk again. Even if your network is different, the network might be a good place to start. Best of luck!

**Comment by pixelauthor** [comment]
Good call on the network issues - that's what it was! I actually resolved it yesterday. Evidently Blackmagic's two tools (the setup tool and the production tool) don't talk to each other. I was changing the IP address in the setup tool, but that wasn't updating the IP address in the other one. I had to change it to the same IP in both programs, and then Companion connected just fine.
